### DISCUSSION TITLE
All My Github Pages are not accessible, it keeps on showing "site can't be reached". How do I solve this issue?

### DISCUSSION THREAD

**Discussion by nattcodes** [community/community#110374] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

All my github pages keeps on returning "This site can't be reached".
This issue started few days ago and I have tried so many things possible. 
Please how do I fix this issue?
My github : https://github.com/nattcodes
My Portfolio site: https://nattcodes.github.io/kome-nath-portfolio/
![Screenshot (38)](https://github.com/community/community/assets/77756722/faf89898-d28d-41b4-b192-50abe9cd1013)
Thank you so much

**Comment by llaravell** [comment]
Hi,
This is it bro:
[kome nath portfolio](https://nattcodes.github.io/kome-nath-portfolio/)

**Reply by nattcodes** [reply]
Thank you so much, all the pages including this, is now active

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @nattcodes ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the GitHub product Discussions. I've gone ahead and moved it for you. Good luck!

**Comment by GarryLowther** [comment]
We also lost our pages web site yesterday. Do we know why Microsoft?

**Reply by yoannchaudet** [reply]
Can you share the link to your site that is inaccessible? Or just the user/org and repo name and I can look it up. I am not aware of any issues on our side but would love to investigate.

**Reply by nattcodes** [reply]
Link: https://nattcodes.github.io/kome-nath-portfolio/
Repo name: kome-nath-portfolio
Thank you so much

**Reply by jfullstackdev** [reply]
it's I think okay by now ? I can see it even in Incognito

![image](https://github.com/community/community/assets/47092464/bcd5a9ae-d16f-4dea-a90f-f555d3a5250d)

**Reply by nattcodes** [reply]
Thank you, I noticed it has to do with my network provider
